### PR TITLE
fix(core): ldml fixes for normalization between transform groups 🙀

### PR DESCRIPTION
**Pull request by srl295**
For #9468

- more complicated marker tests
- tests for any-marker
- fix: proper normalization when strings pass between normalization groups

@keymanapp-test-bot skip

### CODE REVIEW

**Comment by srl295** [comment]
hmm. still passing locally. More work needed! 😆

**Comment by srl295** [comment]
finally a failing test (commented out here).

**Comment by srl295** [comment]
> hmm. still passing locally. More work needed! 😆

i.e. i hadn't found a failing case yet.  Now I have, fixed!  ⚒️

File: core/tests/unit/ldml/test_transforms.cpp [hunk 1]
```diff
@@ -760,6 +760,25 @@ int test_normalize() {
     assert_equal(map[0x0320], 0x1L);
   }
 
+  {
+    // from tests
+    marker_map map;
+    std::cout << __FILE__ << ":" << __LINE__ << "   - complex test 9c" << std::endl;
+    const std::u32string src    = U"9ce\u0300\uFFFF\b\u0002\u0320\uFFFF\b\u0001";
+    const std::u32string expect = U"9ce\uFFFF\b\u0002\u0320\u0300\uFFFF\b\u0001";
```
Comment (mcdurdin) [inline core/tests/unit/ldml/test_transforms.cpp:768]:
```suggestion
    const std::u32string src    = U"9ce\u0300\uFFFF\u0008\u0002\u0320\uFFFF\u0008\u0001";
    const std::u32string expect = U"9ce\uFFFF\u0008\u0002\u0320\u0300\uFFFF\u0008\u0001";
```

**Review by mcdurdin — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.238-alpha](https://keyman.com/downloads/releases/alpha/17.0.238)

**Pull request merged**